drivers/zooz/zooz-zen32-scene-ctlr.groovy

```diff
@@ -396,22 +470,29 @@ List<String> setLED(Number ledNumber, String colorName, brightness) {
          intLevel = 0
    }
    List<String> cmds = []
-   if ((brightness as Integer) <= 0) {
+   if (brightness != null && (brightness as Integer) == 0) {
       // Set LED to "always off" (may want to change in future if add association), unless #5/relay and configured not to:
-      if ((ledNumber as Integer) != 5 || ((ledNumber as Integer) == 5 && (relayLEDBehavior as Integer == 2))) {
-         cmds << zwaveSecureEncap(zwave.configurationV1.configurationSet(scaledConfigurationValue: 2, parameterNumber: ledIndicatorParams[ledNumber as Integer], size: 1))
-      }
-      else {
-         if (enableDebug) log.debug "LED number is 5 and level is 0 but configured to not allow turning off"
+      if (intLedNum != 5 || (relayLEDBehavior as Integer) == 4) {
+         cmds << zwaveSecureEncap(zwave.configurationV1.configurationSet(scaledConfigurationValue: 2, parameterNumber: ledIndicatorParams[intLedNum], size: 1))
+         cmds << zwaveSecureEncap(zwave.configurationV1.configurationGet(parameterNumber: ledIndicatorParams[intLedNum]))
       }
+      else { log.warn "Relay LED (#5) not configured to allow turning off from setLED (no changes made)" }
    }
    else {
-      cmds << zwaveSecureEncap(zwave.configurationV1.configurationSet(scaledConfigurationValue: intColor, parameterNumber: ledColorParams[ledNumber as Integer], size: 1))
-      cmds << zwaveSecureEncap(zwave.configurationV1.configurationSet(scaledConfigurationValue: intLevel, parameterNumber: ledBrightnessParams[ledNumber as Integer], size: 1))
+      cmds << zwaveSecureEncap(zwave.configurationV1.configurationSet(scaledConfigurationValue: intColor, parameterNumber: ledColorParams[intLedNum], size: 1))
+      if (brightness) cmds << zwaveSecureEncap(zwave.configurationV1.configurationSet(scaledConfigurationValue: intLevel, parameterNumber: ledBrightnessParams[intLedNum], size: 1))
       // Set LED to "always on" (may want to change in future if add association):
-      cmds << zwaveSecureEncap(zwave.configurationV1.configurationSet(scaledConfigurationValue: 3, parameterNumber: ledIndicatorParams[ledNumber as Integer], size: 1))
+      if (intLedNum != 5 || (relayLEDBehavior as Integer) == 4) {
+         cmds << zwaveSecureEncap(zwave.configurationV1.configurationSet(scaledConfigurationValue: 3, parameterNumber: ledIndicatorParams[intLedNum], size: 1))
+      }
+      else if (brightness && (relayLEDBehavior as Integer) < 3) { 
+         log.warn "Relay LED (#5) not configured to allow turning on from setLED (skipping 'on' setting)"
+      }
+      cmds << zwaveSecureEncap(zwave.configurationV1.configurationGet(parameterNumber: ledColorParams[intLedNum]))
```

Comment:
Requesting back the state of the settings to track them in the state variables and logs